test/jit/test_class_type.py

```diff
@@ -1027,3 +1027,63 @@ def foo():
             new_list : List[Tuple[float, int, int]] = [(1.0, 1, 1)]
             y.my_list = new_list
             return y
+
+    def test_properties(self):
+        """
+        Test that a scripted class can make use of the @property decorator.
+        """
+        @torch.jit.script
+        class Properties(object):
+            def __init__(self, a: int):
+                self.a = a
+
+            @property
+            def attr(self) -> int:
+                return self.a
```

Comment:
for the getters/setters maybe add some subtraction/addition to stress test this a bit... not that it should rreally matter

Reply:
Done.